test/tests/dampers/interactions/interacting_node_elem_dampers.i

```diff
@@ -0,0 +1,99 @@
+# This model tests interactions between nodal and element dampers.
+# The test verifies that the minimum of the value of a nodal and
+# element damper is always used.
+# If run with the nodal1 and elem1 dampers active, the element damper
+# will govern.  With nodal2 and elem2 dampers, the nodal damper governs.
```

Comment:
Thanks for the comment! I really want to get in the habit of doing this more often.